libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineView.kt

```diff
@@ -531,19 +531,38 @@ class MapboxRouteLineView(options: MapboxRouteLineOptions) {
         }
     }
 
+    /**
+     * Shows the layers used for the primary route line.
+     *
+     * @param style an instance of the [Style]
+     * @param map [MapboxMap] which the routeDrawData should be drawn on
+     * @param callback [RoutesRenderedCallback] to be invoked when the route is rendered
+     */
+    @ExperimentalPreviewMapboxNavigationAPI
+    fun showPrimaryRoute(style: Style, map: MapboxMap, callback: RoutesRenderedCallback) {
```

Comment:
Is this really necessary? The only thing that changes is the layer visibility there's no reapplying of the layer source data.

Reply:
But it still happens async. So we still might run into the situation where we draw incidents without the route line and the line only appears some time after.

Reply:
IMO having one async operation with callback and the other similar one without it is inconsistent from the developer's perspective. I'll leave it as-is for now and look for some app developers' opinions.

Reply:
OK, I wasn't able to. :)
Then I'm gonna stick to the inconsistency argument here.